js/components/wizard_component_modal.jsx

```diff
@@ -17,6 +17,7 @@ function Modal({
   closeModal,
   contentLabel,
   isAlt,
+  isDefinition = true,
```

Comment:
@ddhogan @SirJohn96  Instead of adding this, what if when `title` is empty, the H2 is auto-populated with the icon and "Definition"? Presumably it would be fine for that to have the H2 styling as-is.